build/rule-generator/get-files-metadata.js

```diff
@@ -124,10 +129,13 @@ const getCheckSpecFileMeta = (name, dir) => {
  */
 const getCheckJsFileMeta = (name, dir) => {
 	return {
-		name: `${name}.js`,
+		name: `${name}-evaluate.js`,
 		content: `
 		// TODO: Logic for check
-		return true;
+		function myCheckEvaluate(node) {
```

Comment:
Ditto for here. For a checked named `my-check` the evaluate function should be named `myCheckEvaluate`.